src/pages/GetStarted/GetStartedResults.vue

```diff
@@ -1,16 +1,17 @@
 <template>
 	<div class="get-started-results-page">
-		<kv-progress-bar value="100" max="100" />
+		<kv-progress-bar class="progress-bar" value="100" max="100" />
+
 		<div class="loan-results">
 			<template v-if="totalCount > 0">
 				<h1 class="loan-results__headline">
 					Here {{ countVerb }} {{ countNumber }} {{ countPeople }} you can support!
 				</h1>
 				<p class="loan-results__tagline">
 					In fact, we found {{ totalCount }} {{ countPeople }} that we think you’ll love to lend to.
-					<router-link>
+					<!-- <router-link>
```

Comment:
This throws an error for missing required 'to' prop. Commented out until we figure out where this link should go to.

Reply:
👍